app/ui/build.gradle

```diff
@@ -41,7 +41,8 @@ dependencies {
     implementation "net.jcip:jcip-annotations:1.0"
     implementation "com.jakewharton.timber:timber:${versions.timber}"
     implementation "org.apache.james:apache-mime4j-core:${versions.mime4j}"
-    implementation "org.jetbrains.anko:anko-coroutines:0.10.4"
+    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:1.1.0"
+    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-android:1.1.0"
```

Comment:
1.1.1 is out

Reply:
I know, but this pull reguest is open for a while. We can change this easily.